README.md

```diff
@@ -133,4 +133,4 @@ Keep in mind that there is no set of rules that will satisfy everyone.
 
 ### About ports and names
 The firrtl transformations that result in LoFirrtl alter the names of ports.
-What would be io.a becomes io_a and so forth.
+What would be io.a becomes io_a and so forth.  
```

Comment:
I think is just white space at end of line.
You should run scalafmt on all the files you change.
But I just discovered that the scalafmt plugin is not enabled, will fix that.

Reply:
Actually, for the moment don't worry about scalafmt, it's about to undergo a version change and there's no reason to pollute your PR with formatting stuff.